Packs/CommonWidgets/Widgets/widget-LargestInvestigationsWidget.json

```diff
@@ -0,0 +1,49 @@
+{
+ "id": "LargestInvestigationsWidget",
+ "version": -1,
+ "modified": "2020-08-18T13:28:59.339259+03:00",
+ "name": "Largest Investigations",
+ "dataType": "scripts",
+ "widgetType": "table",
+ "query": "GetLargestInvestigationst",
+ "dateRange": {
+  "fromDate": "0001-01-01T00:00:00Z",
+  "toDate": "0001-01-01T00:00:00Z",
+  "period": {
+   "by": "",
+   "byTo": "",
+   "byFrom": "calendarMonths",
+   "toValue": null,
+   "fromValue": 0,
+   "field": ""
+  },
+  "fromDateLicense": "0001-01-01T00:00:00Z"
+ },
+ "params": {
+  "tableColumns": [
+   {
+    "displayed": true,
+    "isDefault": true,
+    "key": "IncidentID"
+   },
+   {
+    "displayed": true,
+    "isDefault": true,
+    "key": "Size"
+   },
+   {
+    "displayed": true,
+    "isDefault": true,
+    "key": "Date"
+   },
+   {
+    "displayed": true,
+    "isDefault": true,
+    "key": "AmountOfEntries"
+   }
+  ]
+ },
+ "fromVersion": "5.0.0",
```

Comment:
why is this from 5.0?
this feature is for 6.0 and above only

Reply:
done